drone/client.go

```diff
@@ -229,6 +230,15 @@ func (c *client) Build(owner, name string, num int) (*Build, error) {
 	return out, err
 }
 
+// Manually build.
+func (c *client) BuildCreate(owner, name string, branch string, commit string) (*Build, error) {
+	out := new(Build)
+	uri := fmt.Sprintf(pathBuildCreate, c.addr, owner, name)
```

Comment:
You are anyway adding a query value, instead of creating a new route just reuse pathBuilds

Reply:
OK